.env.example

```diff
@@ -3,8 +3,7 @@ DB_PASSWORD=db_password
 DB_NAME=barefoot_nomad
 DB_HOST=127.0.0.1
 DB_TEST_NAME=barefoot_test
-PORT = 
-EMAIL = 
```

Comment:
Why did you remove the `EMAIL, LAYER, PASSWORD, and SECRET_KEY_EMAIL_VERIFY_TOKEN`?